www/modules/custom/borg_project_metrics/borg_project_metrics.module

```diff
@@ -84,4 +90,284 @@ function borg_project_metrics_cron() {
       }
     }
   }
+
+  // Get and store backdrop_project_metrics top level data weekly.
+  if (date('G') == 24) {
+    if (date('l') == 'Monday') {
+      try {
+        $last_date = _get_display_metric_from_db('commits')['date'];
+        _parse_metrics('commits', $last_date);
+      }
+      catch(Exception $e) {
+        $message = $e->getMessage();
+        watchdog('borg_project_metrics', $message, array(), WATCHDOG_ERROR);
+      }
+    }
+
+    if (date('l') == 'Tuesday') {
+      try {
+        $last_date = _get_display_metric_from_db('comments')['date'];
+        _parse_metrics('comments', $last_date);
+      }
+      catch(Exception $e) {
+        $message = $e->getMessage();
+        watchdog('borg_project_metrics', $message, array(), WATCHDOG_ERROR);
+      }
+    }
+  }
+}
+
+/**
+ * Get the number of commits to Backdrop since 15 January 2015.
```

Comment:
yes, but I don't know that date that backdrop started; can you give me an approximate time of the fork?  At least the year?  Unless you know the exact commit that would be great!